articles/communication-services/how-tos/cte-calling-sdk/includes/manage-calls/manage-calls-web.md

```diff
@@ -342,10 +342,26 @@ Consider the following scenario, where Alice makes a call to Bob, then Alice add
 If Teams user stops call recording, the recording is placed into the chat associated with the thread. Provided chat ID impacts the experience of Teams users in Teams clients.
 
 Recommendations for the management of chat ID:
-- Escalation of the 1:1 phone call by adding another phone participant: Use Graph API to get the existing chat ID with only Teams user as a participant or create a new group chat with participants: Teams user ID and "00000000-0000-0000-0000-000000000000"
-- Group call with single Teams user and multiple phone participants: Use Graph API to get existing chat ID with only Teams user as a participant or create a new group chat with participants: Teams user ID and "00000000-0000-0000-0000-000000000000"
-- Group call with more than 2 Teams users: Use Graph API to get or create a group chat with the Teams users
-
+- Escalation of the 1:1 phone call by adding another phone participant:
+  * (Optional way) Use addParticipant API on web calling SDK to add another participant without providing chat thread ID. You only need to provide identifier of the participant you want to add into the call.
+    ```js
+    addParticipant(participant: MicrosoftTeamsUserIdentifier | PhoneNumberIdentifier | MicrosoftTeamsAppIdentifier | UnknownIdentifier)
+    ```
+  * Use Graph API to get the existing chat ID with only Teams user as a participant or create a new group chat with participants: Teams user ID and "00000000-0000-0000-0000-000000000000".
+- Group call with single Teams user and multiple phone participants:
+  * (Optional way) Use startCall API on web calling SDK without providing chat thread ID to create a group chat for participants. You only need to provide identifier for all the participants when leveraging API.
```

Comment:
```suggestion
  * Method `startCall` API allows you to start a group call with multiple participants and optionally provide thread ID. If the parameter isn't provided, then a new group chat is created, and all Microsoft 365 participants are added to the call and chat participants list. If the parameter is provided, then Teams users can see ongoing call associated to this group chat in Teams app. You can create new group chat via Graph API.
```

Reply:
> Is this correct? Do we start group chat or 1:1 chat in the following scenario startCall(M365user,Phone)?

M365 User + Phone , we cannot create a 1:1 chat thread id for this. But in case of group call scenario we will create a chat thread between the M365user and the phone participant.